code/controllers/configuration.dm

```diff
@@ -567,8 +553,9 @@
 				if("irc_announce_new_game")
 					irc_announce_new_game = TRUE
 				else
-					WRITE_FILE(GLOB.config_error_log, "Unknown setting in configuration: '[name]'")
-
+					HandleCommsConfig(name, value)	//TODO: Deprecate this eventually

```

Comment:
add a error behind a #if BYOND_VERSION for 512 so when we move to 512 we are forced to remove this.